products/sle/main.pm

```diff
@@ -572,6 +572,7 @@ sub load_consoletests() {
         loadtest "console/vim.pm";
         if (!is_staging()) {
             loadtest "console/firewall_enabled.pm";
+            loadtest "console/zypper_lifecycle.pm";
```

Comment:
extratests is just stupidly named, but it's a pool of tests where the scenario doesn't matter. Having it in regular console tests is important if storage or software selection makes a difference. This doesn't seem to be the case with this.